### PR TITLE
Fix example_bigtable system test

### PR DESCRIPTION
**Pull request by olegkachur-e**
Fix issues with bigtable system test:

- ensure correct id strings with proper length, as it might be issues depending on length of SYSTEM_TESTS_ENV_ID value.
- assign proper id values to resources used
- add constants for resources names limits

### CODE REVIEW

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 1]
```diff
@@ -63,24 +63,35 @@
 from airflow.utils.trigger_rule import TriggerRule
 from tests.system.providers.google import DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
 
-ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID").lower().replace('_', '-')
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:66]: this line is okay, you don't need to change the content of this variable itself. In every place this stirng is used you can add only 'replace()', without 'lower()'

Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:66]:
```suggestion
ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
```

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 2]
```diff
@@ -63,24 +63,35 @@
...
+
 PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT") or DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
-DAG_ID = "bigtable"
+DAG_ID = f"bigtable"
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:69]: I think we don't need f-string here
```suggestion
DAG_ID = "bigtable"
```

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 3]
```diff
@@ -63,24 +63,35 @@
...
+CB_TABLE_ID_ALLOWED_MAX_LENGTH = 50  # Table-id Length should be between [1,50]
 
-CBT_INSTANCE_ID = f"bigtable-instance-id-{ENV_ID}"
+CB_INSTANCE_ID_BASE = f"inst-id-"
+CB_INSTANCE_ID_BASE = (
+    f"{CB_INSTANCE_ID_BASE}{ENV_ID[:CB_INSTANCE_ID_ALLOWED_MAX_LENGTH-(len(CB_INSTANCE_ID_BASE) + 2)]}"
+)
+CBT_INSTANCE_ID_1 = f"{CB_INSTANCE_ID_BASE}-1"
+CBT_INSTANCE_ID_2 = f"{CB_INSTANCE_ID_BASE}-2"
 CBT_INSTANCE_DISPLAY_NAME = "Instance-name"
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:81]:
```suggestion
CBT_INSTANCE_DISPLAY_NAME = "instance-name"
```

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 4]
```diff
@@ -63,24 +63,35 @@
...
-DAG_ID = "bigtable"
+DAG_ID = f"bigtable"
 
+CB_INSTANCE_ID_ALLOWED_MAX_LENGTH = 33  # Length should be between [6,33]
+CB_CLUSTER_ID_ALLOWED_MAX_LENGTH = 30  # Cluster-id Length should be between [6,30]
+CB_TABLE_ID_ALLOWED_MAX_LENGTH = 50  # Table-id Length should be between [1,50]
 
-CBT_INSTANCE_ID = f"bigtable-instance-id-{ENV_ID}"
+CB_INSTANCE_ID_BASE = f"inst-id-"
+CB_INSTANCE_ID_BASE = (
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:76]: I am afraid it is hard to understand how the name will look like :)

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 5]
```diff
@@ -63,24 +63,35 @@
...
+    f"{CB_INSTANCE_ID_BASE}{ENV_ID[:CB_INSTANCE_ID_ALLOWED_MAX_LENGTH-(len(CB_INSTANCE_ID_BASE) + 2)]}"
+)
+CBT_INSTANCE_ID_1 = f"{CB_INSTANCE_ID_BASE}-1"
+CBT_INSTANCE_ID_2 = f"{CB_INSTANCE_ID_BASE}-2"
 CBT_INSTANCE_DISPLAY_NAME = "Instance-name"
 CBT_INSTANCE_DISPLAY_NAME_UPDATED = f"{CBT_INSTANCE_DISPLAY_NAME} - updated"
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:82]:
```suggestion
CBT_INSTANCE_DISPLAY_NAME_UPDATED = f"{CBT_INSTANCE_DISPLAY_NAME}-updated"
```

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 6]
```diff
@@ -63,24 +63,35 @@
...
 
+CB_INSTANCE_ID_ALLOWED_MAX_LENGTH = 33  # Length should be between [6,33]
+CB_CLUSTER_ID_ALLOWED_MAX_LENGTH = 30  # Cluster-id Length should be between [6,30]
+CB_TABLE_ID_ALLOWED_MAX_LENGTH = 50  # Table-id Length should be between [1,50]
 
-CBT_INSTANCE_ID = f"bigtable-instance-id-{ENV_ID}"
+CB_INSTANCE_ID_BASE = f"inst-id-"
+CB_INSTANCE_ID_BASE = (
+    f"{CB_INSTANCE_ID_BASE}{ENV_ID[:CB_INSTANCE_ID_ALLOWED_MAX_LENGTH-(len(CB_INSTANCE_ID_BASE) + 2)]}"
+)
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:78]:
```suggestion
).replace("_", "-")
```
or vise versa, not sure which format should be here

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 7]
```diff
@@ -63,24 +63,35 @@
...
+CBT_INSTANCE_ID_1 = f"{CB_INSTANCE_ID_BASE}-1"
+CBT_INSTANCE_ID_2 = f"{CB_INSTANCE_ID_BASE}-2"
 CBT_INSTANCE_DISPLAY_NAME = "Instance-name"
 CBT_INSTANCE_DISPLAY_NAME_UPDATED = f"{CBT_INSTANCE_DISPLAY_NAME} - updated"
 CBT_INSTANCE_TYPE = enums.Instance.Type.DEVELOPMENT
 CBT_INSTANCE_TYPE_PROD = 1
 CBT_INSTANCE_LABELS: dict[str, str] = {}
 CBT_INSTANCE_LABELS_UPDATED = {"env": "prod"}
-CBT_CLUSTER_ID = f"bigtable-cluster-id-{ENV_ID}"
+CBT_CLUSTER_ID = f"cluster-id-"
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:87]: Do we need f-string here?

File: tests/system/providers/google/cloud/bigtable/example_bigtable.py [hunk 8]
```diff
@@ -63,24 +63,35 @@
...
+CBT_CLUSTER_ID = f"{CBT_CLUSTER_ID}{ENV_ID[:CB_CLUSTER_ID_ALLOWED_MAX_LENGTH-len(CBT_CLUSTER_ID)]}"
 CBT_CLUSTER_ZONE = "europe-west1-b"
 CBT_CLUSTER_NODES = 3
 CBT_CLUSTER_NODES_UPDATED = 5
 CBT_CLUSTER_STORAGE_TYPE = enums.StorageType.HDD
-CBT_TABLE_ID = f"bigtable-table-id{ENV_ID}"
+CBT_TABLE_ID = f"bigtable-table-id-"
```
Comment (VladaZakharova) [inline tests/system/providers/google/cloud/bigtable/example_bigtable.py:93]: Do we need f-string here?

**Review by VladaZakharova — CHANGES_REQUESTED** [review]
Please, for the task calls please add comments like this 
`(
        #TEST SETUP
        [create_instance_task, create_instance_task2]
        >> create_tables()
        #TEST BODY
        >> wait_for_table_replication_task
        >> update_clusters_and_instance()
        #TEST TEARDOWN
        >> delete_table_task
        >> delete_table_task2
        >> [delete_instance_task, delete_instance_task2]
    )`

**Review by moiseenkov — APPROVED** [review]
Changes look good and the system test is green on staging.

**Comment by olegkachur-e** [comment]
Closing as ready for the community

**Pull request closed without merging**